.github/workflows/r.yml

```diff
@@ -144,7 +144,7 @@ jobs:
           # Don't set a TZ here to test that case. These builds will have the following warning in them:
           #   System has not been booted with systemd as init system (PID 1). Can't operate.
           #   Failed to connect to bus: Host is down
-          archery docker run -e TZ="" r
+          archery docker run -e TZ="" -e GITHUB_ACTIONS="true" r
```

Comment:
Please can you just remind me what this does?

Reply:
Without this `lintr` does not detect being on GHA and no annotation will be generated.